puts "The output of the first is 3. The output of the second is an error due to x variable not initiated in the outside scope."